### ISSUE TITLE
CMM (LMC) Product Video

### ISSUE THREAD

**Issue by Kalpeshdolia** [SSOMENSREPO/MARKETING#5]
@DHANDAPANISATTANATHAN 

SN,

1. Need the CMM (LMC) to be working so that I can complete demo video
2. Following issues need to be resolved to complete User & Admin video:
    a. submission report 
    b. Inventory Site Stock Entry 
    c. Maintain Location under Contract Menu
    d. Approved Contract details when viewing, verification of work done (under superadmin), at the left bottom of page it says : COMPANY: LIH MING CONSTRUCTION PTE LTD, NAME: YAMOAN, this needs to be changed

Pls let me know once rectified...

**Comment by DHANDAPANISATTANATHAN** [comment]
1.Need the CMM (LMC) to be working so that I can complete demo video
 now working fine
2. a. submission report 
     -Will change and give you
b. Inventory Site Stock Entry  - Waiting for client approval.
c. Maintain Location under Contract Menu  - Working 
d. Approved Contract details  - Will change this. Temp mask this.

**Comment by Kalpeshdolia** [comment]
SN,

Status of the following LMC:
2. a. submission report 
-Will change and give you
b. Inventory Site Stock Entry - Waiting for client approval.
d. Approved Contract details - Will change this

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia 

2. a. submission report 
fixed
b. Inventory Site Stock Entry - Waiting for client approval.(Dont show this form in video now) 
d. Approved Contract details - changed 


Close this today.

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

Not able to take video of LMC, still issues with the product

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia 

Stop this. Do HRM user role and EI

**Comment by Kalpeshdolia** [comment]
ok...

**Comment by Kalpeshdolia** [comment]
1. HRM User role video production started
EI not working

**Comment by Kalpeshdolia** [comment]
I recvd http://ei-php-int.appspot.com/ from Sarada, when I open i get the following msg: EI PHP INT
Events from one or more calendars could not be shown here because you do not have the permission to view them.
and NO ACCESS AVAILABLE FOR LOGIN ID : kalman821@gmail.com

**Comment by Kalpeshdolia** [comment]
can't see menu in EI

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia  - Check now EI.

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

EI working now, let me understand the product first, then existing data can be removed...

**Comment by Kalpeshdolia** [comment]
HRM User video done...
https://drive.google.com/drive/folders/0Bxk9WvGwaJX8b3gxdlctVFBoc0E

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

lmc: Report Submission Entry (User) not working

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

LMC: Report Submission Entry (User) not working

@SaradambalMunusamy

EI: Customer creation error: Calendar not created, why?
EI: Deposit deduction calculation error: DOC / SS ID IS WRONG OR CHECK IN CONFIGURATION OR GET ACCESS

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia 

I cannt see the HRM USER VIDEO

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

here is the link...

https://drive.google.com/drive/folders/0Bxk9WvGwaJX8b3gxdlctVFBoc0E

**Comment by Kalpeshdolia** [comment]
@SaradambalMunusamy

EI: Customer creation error: Calendar not created, why? - Status
EI: Deposit deduction calculation error: DOC / SS ID IS WRONG OR CHECK IN CONFIGURATION OR GET ACCESS - Status

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN

LMC: Report Submission Entry (User) not working
          Maintain Location Entry (Admin & Superadmin) not working 

EI: Customer creation error: Calendar not created, why? 
     Deposit deduction calculation error: DOC / SS ID IS WRONG OR CHECK IN CONFIGURATION       OR GET ACCESS

Pls I need working, error free if possible, products to understand and take video...

**Comment by Kalpeshdolia** [comment]
EI: Unit creation working, but newly created unit not available in search

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy

Omega, not loading, demo as well as int

**Comment by Kalpeshdolia** [comment]
Omega, following error:

Error in exception handler: The stream or file "/lamp/demo/omega/app/storage/logs/laravel.log" could not be opened: failed to open stream: Permission denied in /lamp/demo/omega/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:84

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy

Omega, following error:

Error in exception handler: The stream or file "/lamp/demo/omega/app/storage/logs/laravel.log" could not be opened: failed to open stream: Permission denied in /lamp/demo/omega/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:84

**Comment by SaradambalMunusamy** [comment]
**@Kalpesh Dolia**
Now gave user schema performance permission db and ip changed in script both int and dev.
Check now

**Comment by Kalpeshdolia** [comment]
thx

**Comment by Kalpeshdolia** [comment]
@SaradambalMunusamy

Can u pls change logo of omega to Ssomens logo?

**Comment by SaradambalMunusamy** [comment]
**@Kalpesh Dolia**
Updated in int for omega project

**Comment by Kalpeshdolia** [comment]
Thx

**Comment by DHANDAPANISATTANATHAN** [comment]
@SaradambalMunusamy 

In omega - where to see the project name Accounting?

**Comment by SaradambalMunusamy** [comment]
**@DHANDAPANISATTANATHAN**

As per they told i changed and uploaded in int. Didnt inform to put accounting.

Now i updated accounting in omega. In int i uploaded script

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia  & @Archana20 


I cannt see the demo video in GDRIVE. then how to review!

**Comment by Archana20** [comment]
Sir 

Please check now all the latest links are updated in SS.

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM not working...any problem?

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM working now, pw was changed

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM Issues;
1. admin login / employee photo upload error: ErrorException (E_UNKNOWN) 
mkdir(): Permission denied
2. admin login / update job information: both old & new job information are displayed, either the latest should be displayed or there should be a date when the job info was updated
3. Admin login / salary process: unable to change month

for now these 3 issues...

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM Issues:
4. Leave application if applied twice for same dates, AHRM accepts both, it should not. Employee cannot apply twice for the same period.
5. Employee should not be able to apply leave in the past for example: today is 29th Dec 2015, employee should not be able to apply leave for 1st Dec 2015 on 29th dec 2015...
6. Once leave is approved, the number of "Available Balance (Hrs)" should reduce
7. Following error msg comes often: Swift_TransportException 
Connection could not be established with host smtp.mandrillapp.com [Connection timed out #110]

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM Issues:
8. Error in Company Announcement: ErrorException (E_UNKNOWN) 
Undefined index: users
                //echo "<pre>"; print_r($companyDetails);die;            
             $postArray = Input::all(); 
            if(!empty($postArray)) { 
                    if(isset($postArray['send_all']) && $postArray['send_all']=='all') { 
                        $postArray['fkcompany_id'] = 0;
                        $postArray['users'] = "all";
                    } else {
                       $postArray['users'] = implode(",", $postArray['users']);
                    }
                $insertStatus = DB::table('hr_announcements')->insert(array('fkcompany_id' => $postArray['fkcompany_id'],'users' => $postArray['users'],'subject' => $postArray['subject'],'message'=>$postArray['message']));

9. Annual leave is updated in User's HR Manager's login, but not in user's login
10. Sick leave is not updated in User's login as well as HR Manager's Login
11. Not able to view payslip for user in User login

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy 

AHRM Issue:

12. Expiry Reminders not updated in User...

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 
@SaradambalMunusamy

AHRM Issue:

13. In HR Manager login, Scheduled leave report not reflecting employee leave status
14. In User login: Payroll / View Payslip, no information available

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

SN,

**I am not able to complete video recording of the following products due to various issues in the products:**

1. A HRM - 14 issues pending
2. Omega - "omegaadmin" appears as login name, needs to be changed
3. LMC - various issues pending, such as LMC: Report Submission Entry (User) not working / Maintain location Entry not working. You asked me to stop as customer details/replies are awaited.
4. EI - need empty db

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia ,

1.A HRM - record what r the forms working now that is enough.
2.omega - no issue in login name. record video
3.LM C - ALREADY I TOLD TO YOU RECORD THOSE WORKING ONLY.
4.EI - YOU RECORD W/O EMPTY DB, BUT DONT SHOW THE EI CUSTOMER NAME AND DETAILS.

I HOPE YOU LL CLOSE ALL THESE TMRW.

I CANNT POSTPONE THIS. TMRW I SHLD LAUNCH SSO WEBSITE.

WE CAN SHOW FULL DEMO WHEN WE GET CLIENT.

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

All my questions regarding video recording were for Product video and not for new ssomens website. For new website we have screen shots and the product videos can be linked once they are ready...

**Comment by DHANDAPANISATTANATHAN** [comment]
@Kalpeshdolia,

Complete and deliver all product video. if any issue that part only dont record. Record each form separately. We shld launch SSO WEBSITE 1ST JAN 2016.

LATER WILL JOIN THOSE ISSUES FORMS ONCE RECTIFIED.

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

ok...

**Comment by Kalpeshdolia** [comment]
@DHANDAPANISATTANATHAN 

1. A HRM - User roll completed (some form unable to take video)
2. Omega - To start
3. LMC - User roll completed (some form unable to take video) 
4. EI - In Process
